### DISCUSSION TITLE
How do I set the CLIPBOARD_ALWAYS_PERSIST environmental variable???

### DISCUSSION THREAD

**Discussion by BenjaminWang236** [Slackadays/Clipboard#65] [Q&A] [answered]
In the configuration `CLIPBOARD_ALWAYS_PERSIST   Set this to make Clipboard always use persistent clipboards.` what exactly does `Set this` mean?

Since this is environmental variable, I assume I would be doing the standard `export CLIPBOARD_ALWAYS_PERSIST=...` but what do I set it equal to?

**Comment by Slackadays** [comment, marked as answer]
You didn't read `Set this` wrong, it literally means that if the variable `is set` to something, anything, then it's considered enabled to Clipboard. So, in your case, use `export CLIPBOARD_ALWAYS_PERSIST=1` or `...ERSIST=0`, as Clipboard doesn't check what the value actually is. That also means to disable the variable, just `unset` it.

I'll update the readme to say `Set this to anything...` instead so it's clearer :)
